app/src/features/apiClient/screens/apiClient/components/clientView/APIClientView.tsx

```diff
@@ -788,14 +805,20 @@ const APIClientView: React.FC<Props> = ({
               hotKey={KEYBOARD_SHORTCUTS.API_CLIENT.SEND_REQUEST.hotKey}
               type="primary"
               className="text-bold"
-              disabled={!entry.request.url}
+              disabled={
+                !entry.request.url ||
+                (appMode === "EXTENSION" && entry.request.contentType === RequestContentType.MULTIPARTFORM)
+              }
```

Comment:
While request is going on currently we don't disable the `Send` button, ideally should be disabled, confirm this with product. 
If there are multiple send clicks, response view will be updated async, which will be broken UX.